packages/cyberstorm/src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,59 @@
+"use client";
+import React, { CSSProperties, ReactNode } from "react";
+import styles from "./Toast.module.css";
+import { Icon } from "../Icon/Icon";
+import { classnames } from "../../utils/utils";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faXmarkLarge } from "@fortawesome/pro-solid-svg-icons";
+import * as RadixToast from "@radix-ui/react-toast";
+
+type _ToastProps = {
+  id: string;
+  content: ReactNode;
+  icon?: JSX.Element;
+  variant?: "info" | "danger" | "warning" | "success";
+  timer?: number;
+};
+export type ToastProps = _ToastProps &
+  Omit<React.HTMLProps<HTMLDivElement>, keyof _ToastProps>;
```

Comment:
This comment is outside the scope of the PR since the approach used here is copy-pasted all over the code base already. I was planning to address this myself but that was a month ago and it seems I can't find the time for it. Maybe you can add this to your TODO list?

Anyways, found this article, which in the collapsible "Why not..." box describes the shortcomings of this approach (and the main article offers a preferred alternative): https://react-typescript-cheatsheet.netlify.app/docs/advanced/patterns_by_usecase/#maybe-reactjsxintrinsicelements-or-elementhtmlattributes

Reply:
I'll add this to the list of stuff to do 👍 TS-1965